systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java

```diff
@@ -39,12 +39,17 @@
     }
 
     public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas) {
-        return kafkaConnect(name, name, kafkaConnectReplicas);
+        return kafkaConnect(name, name, kafkaConnectReplicas, true);
     }
 
-    public static DoneableKafkaConnect kafkaConnect(String name, String clusterName, int kafkaConnectReplicas) {
+    public static DoneableKafkaConnect kafkaConnect(String name, int kafkaConnectReplicas, boolean allowNP) {
+        return kafkaConnect(name, name, kafkaConnectReplicas, allowNP);
+    }
+
+    public static DoneableKafkaConnect kafkaConnect(String name, String clusterName, int kafkaConnectReplicas, boolean allowNP) {
```

Comment:
You can remove the booleans similarly from here and line 45

Reply:
Ah leftover ... sorry Sam

Reply:
Hmmm ... I was thinking that I'll leave it like this, so we just specify in the KafkaConnect "creation" if we want it with NP or not..

But yes, I can create the `kafkaConnectWithoutNetworkPolicies` method and `kafkaConnectWithNetworkPolicies`.

These two (which you mentioned - line 45 and 49) are global and if I will change the naming, I will have to change it everywhere.

WDYT @samuel-hawker and @Frawless ? Will we leave it like this or should I rename them all?

Reply:
I don't think it's that problematic here, since this is the highest level of abstraction I think we are going when deploying a KafkaConnect.
I am happy for you to leave as is, but there is nothing worse than passing a boolean through a chain of 5 functions was my origin point! (Which this isn't doing)

Reply:
Maybe it's really ridiculous, but I have some changes (small refactor) of these Network policies methods for all other resources. How about to do everything in that one? Sorry ... I'll request you for the review there

Reply:
No problem!